src/javascript/binary/websocket_pages/user/account/settings.js

```diff
@@ -20,6 +20,10 @@ const Settings = (() => {
                 $('#change_password').removeClass(class_hidden);
             }
 
+            if (!response.get_account_status.prompt_client_to_authenticate) {
+                $('#authenticate').addClass(class_hidden);
```

Comment:
What about `removeClass` and reverse condition? as it's already invisible in the template.